tools/compare.py

```diff
@@ -244,14 +250,22 @@ def main():
         json2 = gbench.report.filter_benchmark(
             json2_orig, filter_contender, replacement)
 
-    # Diff and output
-    output_lines = gbench.report.generate_difference_report(
-        json1, json2, args.display_aggregates_only,
+    # Diff and output to stdout
+    output_json = gbench.report.get_difference_report(
```

Comment:
It's not json per-se, just some data structure that isn't the console output

Reply:
Agreed. Done.